packages/arb-token-bridge-ui/src/components/TransactionHistory/TransactionsTableRow.tsx

```diff
@@ -178,32 +189,55 @@ export function TransactionsTableRow({
     >
       <div className="pr-3 align-middle">{txRelativeTime}</div>
       <div className="flex items-center pr-3 align-middle">
-        <TransactionsTableTokenImage tx={tx} />
-        <span className="ml-2">
-          {formatAmount(Number(tx.value), {
-            symbol: tokenSymbol
-          })}
-        </span>
+        <TransactionsTableExternalLink
+          href={`${getExplorerUrl(
+            tx.isWithdrawal ? tx.childChainId : tx.parentChainId
+          )}/token/${tx.tokenAddress}`}
+          disabled={!tx.tokenAddress}
+        >
+          <TransactionsTableTokenImage tx={tx} />
+          <span className="ml-2">
+            {formatAmount(Number(tx.value), {
+              symbol: tokenSymbol
+            })}
+          </span>
+        </TransactionsTableExternalLink>
       </div>
       <div className="flex items-center space-x-2">
-        <span>
-          <NetworkImage
-            chainId={tx.isWithdrawal ? tx.childChainId : tx.parentChainId}
-          />
-        </span>
-        <span className="inline-block w-[55px] break-words">
-          {getNetworkName(tx.isWithdrawal ? tx.childChainId : tx.parentChainId)}
-        </span>
+        <TransactionsTableExternalLink
+          href={`${getExplorerUrl(
+            tx.isWithdrawal ? tx.childChainId : tx.parentChainId
```

Comment:
this has been repeating, maybe move it to a var?

Reply:
updated